package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@rebilly/framepay-react",
-    "version": "1.5.1",
+    "version": "1.6.0",
```

Comment:
Lets not bump the version, because there are no changes to the build of the library, so the version bump wont follow semantic versioning

Reply:
Reverted back to 1.5.1! Thank you for the comment!